README.md

```diff
@@ -4,140 +4,136 @@
 [![Carthage compatible](https://img.shields.io/badge/Carthage-compatible-4BC51D.svg?style=flat)](https://github.com/morizotter/TouchVisualizer)
 [![Circle CI](https://circleci.com/gh/morizotter/TouchVisualizer/tree/master.svg?style=shield&circle-token=b7eb2e179731634bcac95d1e4f8e90b837b092e3)](https://circleci.com/gh/morizotter/TouchVisualizer/tree/master) [![Join the chat at https://gitter.im/morizotter/TouchVisualizer](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/morizotter/TouchVisualizer?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-TouchVisualizer is a lightweight and pure Swift implemented library for visualizing touches on the screen. Let's give an effective presentation with TouchVisualizer!
-
-## Features
-
-- Works with **just a single line of code**!
-- Multiple fingers supported.
-- Multiple UIWindows supported.
-- Shows touch radius.
-- Shows touch duration.
-- You can change colors and images of finger points.
-- iPhone and iPad with portlait and landscape supported.
-
-## How it looks
-
-### Portlait
-
+TouchVisualizer is a lightweight pure-Swift implementation for visualising `UITouch` events.
```

Comment:
`touches on the screen` is more appeal than `UITouch` because `UITouch` is a little bit detail